payment-request/allowpaymentrequest/removing-allowpaymentrequest.https.sub.html

```diff
@@ -13,7 +13,7 @@
   const path = location.pathname.substring(0, location.pathname.lastIndexOf('/') + 1);
   iframe.src = "https://{{domains[www1]}}:{{ports[https][0]}}" + path + "echo-PaymentRequest.html";
   iframe.onload = t.step_func(() => {
-    if (i === 0) {
+    if (i === 1) {
```

Comment:
I'm confused. When `onload` is called, the exception is still that `i` should be 0. Otherwise `iframe.allowPaymentRequest = false;` is never set? I think the same applies for the rest? 

@zcorpan, you wrote these tests, right? Can you confirm?